src/wazuh_modules/vulnerability_detector/wm_vuln_detector_nvd.c

```diff
@@ -1821,6 +1828,12 @@ int wm_vuldet_find_nvd_vulnerabilities(sqlite3 *db, agent_software *agent, wm_vu
             raw_version = (char *)sqlite3_column_text(stmt, 12);
             raw_arch = (char *)sqlite3_column_text(stmt, 13);
 
+            // If there's no MSU data, disable the hotfix scan.
+            if (stored_cpe->part[0] == 'o' && wm_vuldet_check_enabled_msu(db) == false) {
+                mtdebug1(WM_VULNDETECTOR_LOGTAG, VU_NO_HOTFIX_AVAIL, atoi(agent->agent_id));
```

Comment:
I'd trigger a warning here. To have the OS CPE indexed means the hotfixes scan is enabled (see `wm_vuldet_get_software_info`) so this is unexpected behavior. In addition, use a new log instead of `VU_NO_HOTFIX_AVAIL`, it would be useful to explain that the MSU database is empty.

Reply:
Done